content/docs/get-started/azure/modify-program.md

```diff
@@ -64,14 +64,14 @@ EOT
 
 {{% /choosable %}}
 
-Now that you have your new `index.html` with some content, you can enable static website support, upload `index.html` to a storage container, and retrieve a public URL through the use of resource properties. These properties can be used to define dependencies between related resources or to retrieve property values for further processing.
-
-To see how this all works, open the program file in your IDE or text editor and add the following the lines right after the storage account creation.
+Now that you have your new `index.html` with some content, enable static website support, upload `index.html` to a storage container, and retrieve a public URL through the use of resource properties. These properties can be used to define dependencies between related resources or to retrieve property values for further processing.
```

Comment:
I think this reads much more clearly with the "you can" it as before, since this paragraph is about listing out what I'm going to do next. But I defer to you -- I just had to read it a couple of times before I understood what it was saying.